eclipse/settings/README.md

```diff
@@ -0,0 +1,77 @@
+# Project Settings for Eclipse
+
+This project contains the canonical set of Eclipse `.settings/`
+files for Cloud Tools for Eclipse.  These settings configure Eclipse
+to use the [Google Style for Java][google-style-java].
+
+The settings currently reflect the following choices:
+
+  - uses the [Google Java Format plugin for Eclipse](google-java-format),
+    which requires Eclipse Oxygen (4.7) or later.
+  - unnecessary declared exceptions on methods are _errors_
+  - switch missing default case are _warnings_
+  - resource leaks are _errors_
+  - potential resource leaks are _warnings_
+  - serializable classes without `serialVersionUID` are _ignored_
+  - redundant generic type argument are _errors_
+
+## Updating the Settings
+
+The settings require installing the [Google Java Format plugin for
+Eclipse][google-java-format].  Installing the format plugin requires
+downloading the [latest release][google-java-format-release] (named
+`google-java-format-eclipse-plugin_XXXX.jar`)and placing the jar
+in your Eclipse installation's `dropins/` directory (on MacOS, in
+`Eclipse.app/Content/Eclipse/dropins/`).
+
+Updating and applying the settings files is a two-step process based
+around the [`eclipse-settings-maven-plugin`][esmp] Maven plugin.
+
+  [esmp]: https://github.com/BSI-Business-Systems-Integration-AG/eclipse-settings-maven-plugin
```

Comment:
I prefer direct inline links but up to you.

Reply:
Me too, but the text was becoming hard to read :-/